tools/wpt/browser.py

```diff
@@ -379,7 +379,7 @@ def _latest_geckodriver_version(self):
         tags = call("git", "ls-remote", "--tags", "--refs",
                     "https://github.com/mozilla/geckodriver.git")
         release_re = re.compile(r".*refs/tags/v(\d+)\.(\d+)\.(\d+)")
-        latest_release = 0
+        latest_release = [0]
```

Comment:
Line 389 also needs changed, as it checks whether latest release is equal to `0`.